src/main/java/com/google/sps/servlets/utils/VideoInformation.java

```diff
@@ -24,7 +24,7 @@ public VideoInformation(VideoListResponse videoResponse) {
     VideoStatistics videoStats = video.getStatistics();
     this.videoName = videoSnippet.getTitle();
     this.videoAuthor = videoSnippet.getChannelTitle();
-    this.publishDateString = videoSnippet.getPublishedAt().toStringRfc3339();
+    this.publishDateString = videoSnippet.getPublishedAt().toStringRfc3339().substring(0, 10);
```

Comment:
What is this change for?

Reply:
This removes the unreadable code at the end of the dateString. Making it yy-mm-dd format

Reply:
I also feel that it's ideal to have this change on the front end, but making the change here is fine.

The primary reason why it would be useful to have any date display logic on the front end is because the front end should be able to determine the time zone of the end-user and be able to display the date/time accordingly (which the backend can't do).

But can you add a comment here indicating why you're doing this?

Reply:
Agreed with Rick.